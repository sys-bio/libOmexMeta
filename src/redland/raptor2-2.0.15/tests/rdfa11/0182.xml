<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <title>Test 0182</title>
        <base href="http://example.org/"/>
    </head>
    <body>
        <div prefix="foaf: http://example.org/wrong/foaf/uri/ dc: http://purl.org/dc/terms/">
            <div about="#this" typeof="dc:Agent" prefix="foaf: http://xmlns.com/foaf/0.1/">
                <p property="foaf:name">A particular agent</p>
            </div>
        </div>
    </body>
</html>