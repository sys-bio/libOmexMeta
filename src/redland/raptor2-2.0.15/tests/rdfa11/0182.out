<http://example.org/#this> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/dc/terms/Agent> .
<http://example.org/#this> <http://xmlns.com/foaf/0.1/name> "A particular agent" .
